### ISSUE TITLE
Use basic form component

### ISSUE THREAD

**Issue by burotica49** [jeroennoten/Laravel-AdminLTE#866]
Hello,

I'm trying to use the Form components but I must have forgotten a step.
I have integrated in my view "<x-adminlte-input name =" username "id =" userid "value =" 100 "size =" 5 "label =" Amount "/>" but I have an error "Unable to locate a class or view for component [{$ component}]. "

Thank you for your help

LARAVEL 8
Laravel Breeze
Laravel-AdminLTE 3.5.4

**Comment by Shidersz** [comment]
@burotica49 There is no release with components. They are in development and we expect to launch they on the next release candidate. You can use the `dev-master` tag in your `composer.json` file to experiment with they. But take note they are in development stage.

```json
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": [
        "framework",
        "laravel"
    ],
    "license": "MIT",
    "require": {
        "php": "^7.3",
        "fideloper/proxy": "^4.2",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "jeroennoten/laravel-adminlte": "dev-master", <==== HERE!
        "laravel/framework": "^8.0",
        "laravel/tinker": "^2.0",
        "laravel/ui": "^2.0"
    },
    ...
```

After editing `composer.json` you need to run `composer update` command.

**Comment by Shidersz** [comment]
Components will be available for package version **v3.6.0** or greater.
